app/imports/components/projects/AddProject.jsx

```diff
@@ -133,6 +185,16 @@ export class AddProject extends Component {
     this.setState(this.getInitalState());
   }
 
+  callAddProject(project) {
+    addProject.call(project, (err, res) => {
+      if (err) {
+        console.error(err.error);
+      } else {
+        this.cleanForm();
```

Comment:
this should take you to either the single route for the created project or the list of projects